### PR TITLE
[16.0][MIG] partner contact age range

### PR DESCRIPTION
**Pull request by Highcooley**
Migrated to 16.0
Improved _compute_age_range_id method
Added Unit test for age_range = False case

### CODE REVIEW

**Comment by Highcooley** [comment]
My first code contribution to the OCA. Thank you for your reviews.

**Review by cyrilmanuel — APPROVED** [review]
not an expert but LGTM

**Comment by Highcooley** [comment]
@rousseldenis could you help me out with the nice migration and needs review label and maybe a review itself, please? Thanks a lot!

File: partner_contact_age_range/models/res_partner.py [hunk 1]
```diff
@@ -19,18 +19,15 @@ def _compute_age_range_id(self):
...
-                    )
-                    or False
+                age_range = age_ranges.filtered(
+                    lambda age_range: age_range.age_from
+                    <= record.age
+                    <= age_range.age_to
                 )
             else:
-                age_range = False
+                age_range = self.env["res.partner.age.range"]
```
Comment (rousseldenis) [inline partner_contact_age_range/models/res_partner.py]: `self.env["res.partner.age.range"].browse()` is better

**Review by rousseldenis — APPROVED** [review]
Minor change

**Review by mmequignon — APPROVED** [review]
LG!
Can you please address @rousseldenis 's comment?

**Comment by Highcooley** [comment]
Done

**Pull request merged**